src/types/positions.ts

```diff
@@ -28,7 +28,13 @@ export interface NFTData {
   name: string;
 }
 
-export type Permission = 'INCREASE' | 'REDUCE' | 'WITHDRAW' | 'TERMINATE';
+export enum Permission {
+  increase = 'INCREASE',
```

Comment:
May I suggest using ALL_CAPS for the enum entries? I think that's the convention

Reply:
sure